src/installing-workflow/system-requirements.md

```diff
@@ -1,34 +1,45 @@
-# Resource Requirements
+# Requirements
 
-When deploying Deis, it's important to provision machines with adequate resources. Deis is a highly-available distributed system, which means that Deis components and your deployed applications will move around the cluster onto healthy hosts as hosts leave the cluster for various reasons (failures, reboots, autoscalers, etc.). Because of this, you should have ample spare resources on any machine in your cluster to withstand the additional load of running services for failed machines.
+If you are planning to run Deis Workflow on your own Kubernetes cluster there are a few requirements that you should be
+aware of when creating your environment.
 
-## Resources
+## Resource Requirements
 
-Deis components consume approximately 2 - 2.5GB of memory across the cluster, and approximately 30GB of hard disk space. Because each machine should be able to absorb additional load should a machine fail, each machine must have:
+When deploying Deis Workflow, it's important to provision machines with adequate resources. Deis is a highly-available
+distributed system, which means that Deis components and your deployed applications will move around the cluster onto
+healthy hosts as hosts leave the cluster for various reasons (failures, reboots, autoscalers, etc.). Because of this,
+you should have ample spare resources on any machine in your cluster to withstand the additional load of running
+services for failed machines.
+
+Deis Workflow components consume approximately 2 - 2.5GB of memory across the cluster, and approximately 30GB of hard
```

Comment:
Some repetition here. Maybe:
> Deis Workflow components use about 2.5GB of memory across the cluster, and require approximately 30GB of hard disk space. Because it may need to handle additional load if another one fails, the minimum requirements for each machine are: